decompiler/structures/pointers.py

```diff
@@ -80,7 +83,7 @@ def _add_pointers_without_aliased_variables(self, instr: Instruction):
         the points-to with empty set of associated variables
         """
         for var in instr.requirements:
-            if isinstance(var, Variable) and isinstance(var.type, Pointer):
+            if (isinstance(var, Variable) or isinstance(var, GlobalVariable)) and isinstance(var.type, Pointer):
```

Comment:
Since `GlobalVariable` inherits from `Variable`, this change should not change semantics

Reply:
Is debugging leftover, will remove.

Reply:
Removed.